types/setting.go

```diff
@@ -356,3 +358,30 @@ func GetCustomizedDefaultSettings() (map[string]string, error) {
 
 	return defaultSettings, nil
 }
+
+func OverwriteBuiltInSettingsWithCustomizedValues() error {
+	if LoadedCustomizedDefaultSettings {
```

Comment:
Not sure how this work.
```
The boolean variable `LoadedCustomizedDefaultSettings` is added
to indicate that all the default settings are overwritten by the
customized values. Then some daemon pods don't need to read the
file and overwrite the settings.
```

Can this variable help other pods?

Reply:
Okay, it's useless...